### DISCUSSION TITLE
* how to resolve vscode forge run

### DISCUSSION THREAD

**Discussion by Ricardo-MUZ** [microsoft/vscode-jupyter#13381] [Questions and Answers]
(*) error forge in vscode, using firebase functions in VM local. Know yeah wich using css and markdown and extensions.

/npm install not working too

<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/25423296/163456776-7f95b81a-f1ed-45f7-b7ab-8fa810d529fa.png">
  <source media="(prefers-color-scheme: light)" srcset="https://user-images.githubusercontent.com/25423296/163456779-a8556205-d0a5-45e2-ac17-42d089e3c3f8.png">
  <img alt="Shows an illustrated sun in light mode and a moon with stars in dark mode." src="https://user-images.githubusercontent.com/25423296/163456779-a8556205-d0a5-45e2-ac17-42d089e3c3f8.png">
</picture>

**Comment by DonJayamanne** [comment]
Your question doesn’t seem to apply to Jupyter, it seems very generic
 Please file an issue against vscode,
